internal/cli/atlas_alert_config.go

```diff
@@ -25,6 +25,8 @@ func AtlasAlertConfigBuilder() *cobra.Command {
 		Short:   "Manage Atlas alert configurations for your project.",
 		Long:    "The alertConfigs command provides access to your alerts configurations. You can create, edit, and delete alert configurations.",
 	}
+
+	cmd.AddCommand(AtlasAlertConfigCreateBuilder())
```

Comment:
this is not your fault but mine as I designed the naming, but after checking out your branch and testing it I'm not convince about `mcli atlas alert-configs cmd` structure, I think we may change this to be `mcli atlas alerts configs cmd` and let it leave next to the `mcli atlas alerts cmd`